### PR TITLE
Use psutil where possible in win_status.py

### PR DESCRIPTION
**Pull request by twangboy**
### What does this PR do?
Uses PSUTIL where possible to simplify and fix broken code
- `status.cpuload` failed if more than 1 cpu was present

### What issues does this PR fix or reference?
https://github.com/saltstack/salt/issues/41428

### Tests written?
No

### CODE REVIEW

File: salt/modules/win_status.py [hunk 1]
```diff
@@ -12,26 +12,24 @@
 
 # Import Python Libs
 from __future__ import absolute_import
-import os
-import ctypes
-import sys
-import time
 import datetime
+import psutil
```
Comment (cachedout) [inline salt/modules/win_status.py]: This will stacktrace. We only have `psutil` as a dep for Nitrogen.

**Review by cachedout — CHANGES_REQUESTED** [review]

File: salt/modules/win_status.py (hunk 1 above)
Comment (twangboy) [inline salt/modules/win_status.py]: Gated the import

**Review by cachedout — APPROVED** [review]

**Comment by terminalmage** [comment]
Wouldn't this add a runtime dep on psutil? If so, this is probably not a huge deal as we have a lot better control over the modules we distribute on Windows.

**Comment by twangboy** [comment]
We're already packaging psutil with Windows.

**Comment by terminalmage** [comment]
Yes, I kinda figured we were (or could do so easily).

**Pull request merged**